registrations/serializers.py

```diff
@@ -98,7 +119,9 @@ def get_signups(self, obj):
                 queryset = SignUp.objects.filter(registration__id=obj.id)
                 return SignUpSerializer(queryset, many=True, read_only=True).data
             else:
-                return f"Only the admins of the organization that published the event {event.id} have access rights."
+                return _(
+                    f"Only the admins of the organization that published the event {event.id} have access rights."
+                )
```

Comment:
Unfortunately gettext does not support f-strings, so this needs to be something like
```python
return _(
    "Only the admins of the organization that published the event {event_id} have access rights."
).format(event_id=event.id)
```

Reply:
Changed. f-string was used in gettext in other place too. Fixed there as well.